docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -9,6 +9,22 @@ If your problem isn’t described here, please review open issues in the followi
 
 Have a question? Contact us in the https://discuss.elastic.co/[discuss forum].
 
+[float]
+[[rule-fails-to-decrypt-api-key]]
+=== Failing to decrypt apiKey
+
+*Problem*:
+
+The rule fails to execute and has an `Unable to decrypt attribute "apiKey"` error.
+
+*Solution*:
+
+This error happens when the `xpack.encryptedSavedObjects.encryptionKey` value changed since creating the rule. Depending on the scenario, there are different ways to solve this problem:
```

Comment:
This describes the problem in scenario 1, but not necessarily scenario 2. Maybe a more general description of the problem would be something like `This error happens the `xpack.encryptedSavedObjects.encryptionKey` value used to create the rule does not match the value used during rule execution`?

Reply:
++ I went ahead and changed the wording.